src/styles/variables.js

```diff
@@ -46,5 +48,14 @@ export default {
     tooltipzIndex: 10050,
     gutterWidth: 16,
     popoverMenuShadow: '0px 4px 12px 0px rgba(0, 0, 0, 0.06)',
+    lineHeightSmall: PixelRatio.getFontScale() * 14 > 16 ? 16 : 14 * PixelRatio.getFontScale(),
+    lineHeightNormal: PixelRatio.getFontScale() * 16 > 21 ? 21 : 16 * PixelRatio.getFontScale(),
+    lineHeightLarge: PixelRatio.getFontScale() * 18 > 24 ? 24 : 18 * PixelRatio.getFontScale(),
+    lineHeightXLarge: PixelRatio.getFontScale() * 20 > 22 ? 22 : 20 * PixelRatio.getFontScale(),
+    inputHeight: PixelRatio.getFontScale() * 52 > 72 ? 72 : 52 * PixelRatio.getFontScale(),
+    pickerIconTop: PixelRatio.getFontScale() * 16 > 26 ? 26 : 16 * PixelRatio.getFontScale(),
+    formErrorLineHeight: PixelRatio.getFontScale() * 18 > 23 ? 23 : 18 * PixelRatio.getFontScale(),
+    communicationsLinkHeight: PixelRatio.getFontScale() * 20 > 30 ? 30 : 20 * PixelRatio.getFontScale(),
+    alternateTextHeight: PixelRatio.getFontScale() * 20 > 22 ? 22 : 20 * PixelRatio.getFontScale(),
```

Comment:
Don't we want them to be dynamic?

Reply:
Yes, we want them because it make it possible to decrease the App `fontSize` and see multiple zoom levels rather than default and max.

Reply:
To clarify, this does not appear to be dynamic - we are exporting constant values and must use functions to get the current font scale? Does that sound correct?

Reply:
Yes, we must use functions in order to get the current font scale, but this will make a background to foreground changes reflection as in [this video](https://user-images.githubusercontent.com/51829206/171215944-87b9f7d3-aea0-4919-8977-e8e58011b57f.mp4) and we don’t need this because there is a bug there.

Reply:
What is meant by "a background to foreground changes reflection"? Can you say that another way? I also don't understand what "bug" you're referring to.

Reply:
@osama256 Can you rephrase your comment.

Reply:
@marcaaron "a background to foreground changes reflection" means changing the device font size while the App is in the background (not completely closed) and then resume the App.

>I also don't understand what "bug" you're referring to.

the "bug" is : 

1. on ios devices, If you open the App using default device font size you will get result 1.
2. if you open the App using larger device font size, send the App to the background, switch to the default device font size and resume the App you will get result 2.
3. Reault 1 should be the same as result 2 but this not happening as in [this video](https://user-images.githubusercontent.com/51829206/171215944-87b9f7d3-aea0-4919-8977-e8e58011b57f.mp4), this is the bug.